src/internet_identity/src/assets.rs

```diff
@@ -134,3 +142,57 @@ fn get_assets() -> [(&'static str, &'static [u8], ContentEncoding, ContentType);
         ),
     ]
 }
+
+pub fn increase_asset_counter(asset: String, status_code: u16) {
+    const MAX_TRACKED_ASSETS: u8 = 30;
+
+    if (200..400).contains(&status_code) {
```

Comment:
this looks like it checks for the following:

``` rust
if status_code >= 200 && status_code < 400 {
```

although the intent is not clear, and without looking at the doc I can't tell whether `400` is included in the range. Although, this seems orders of magnitude slower... so surely I'm missing something?

Reply:
I made #1228 to disable the lint.